src/eventEmitter/index.js

```diff
@@ -55,4 +70,53 @@ myEmitter.on('delete-exercise', (exercise, user) =>
   createDeleteExerciseEvent(exercise, user)
 )
 
+// CURRICULUM
+myEmitter.on('create-curriculum', (curriculum, user) =>
```

Comment:
Split this file up so functions/events relating to curriculums, modules, units, etc are all in their own files.